connect/api/src/main/java/org/apache/kafka/connect/data/Struct.java

```diff
@@ -229,7 +229,12 @@ public void validate() {
             Object value = values[field.index()];
             if (value == null && (fieldSchema.isOptional() || fieldSchema.defaultValue() != null))
                 continue;
-            ConnectSchema.validateValue(fieldSchema, value);
+            try {
+                ConnectSchema.validateValue(fieldSchema, value);
+            } catch(DataException e) {
+                throw new DataException("Validate failed for required field: \""  + field.name() + "\", "
```

Comment:
Throwing new DataException(..., e) doesnet include original exception message, but only set up original exception as the cause in new exception. @lindong28